src/main/java/com/google/devtools/build/lib/windows/jni/WindowsFileOperations.java

```diff
@@ -66,7 +66,6 @@ private WindowsFileOperations() {
   private static final int DELETE_PATH_ACCESS_DENIED = 4;
 
   private static final int READ_SYMLINK_OR_JUNCTION_SUCCESS = 0;
-  // READ_SYMLINK_OR_JUNCTION_ERROR = 1;
   private static final int READ_SYMLINK_OR_JUNCTION_ACCESS_DENIED = 2;
```

Comment:
Nit: maybe 1, 2, 3, 4,?

Reply:
These values match constants in `src/main/native/windows/file.h`. Added comments.